tools/openapi2jsonschema/examples/airbyte.local/openapi.yaml

```diff
@@ -1260,6 +1260,27 @@ paths:
       summary: Update a connection
       tags:
         - web_backend
+  /v1/web_backend/connections/updateNew:
```

Comment:
Even though this should be temporary, does it make sense to name it `updatePerStream` rather than `new`?

Reply:
Is it ok if we leave as is just to get this in?

Reply:
yep, forgot to tap the comment as optional